lib/template.html

```diff
@@ -1,5 +1,7 @@
 <!-- Begin Jekyll SEO tag v{{ seo_tag.version }} -->
 
+{% assign seo_page_url = page.url | replace:'/index.html','/' | absolute_url %}
+
 {% if page.url == "/" or page.url == "/about/" %}
```

Comment:
I guess it's because the default `page.url` for a Jekyll site `index.html` is `/index.html`